x-pack/plugins/security_solution/public/management/components/console/components/command_list.test.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ConsoleTestSetup, HelpSidePanelSelectorsAndActions } from '../mocks';
+import {
+  getCommandListMock,
+  getConsoleTestSetup,
+  getHelpSidePanelSelectorsAndActionsMock,
+} from '../mocks';
+import React from 'react';
+import { waitFor } from '@testing-library/react';
+
+describe('When rendering the command list (help output)', () => {
+  let render: ConsoleTestSetup['renderConsole'];
+  let renderResult: ReturnType<typeof render>;
+  let consoleSelectors: ConsoleTestSetup['selectors'];
+  let enterCommand: ConsoleTestSetup['enterCommand'];
+
+  beforeEach(() => {
+    const testSetup = getConsoleTestSetup();
+
+    render = (props = {}) => (renderResult = testSetup.renderConsole(props));
+    consoleSelectors = testSetup.selectors;
+    enterCommand = testSetup.enterCommand;
+  });
+
+  describe('and its displayed on the side panel', () => {
+    let renderAndOpenHelpPanel: typeof render;
+    let helpPanelSelectors: HelpSidePanelSelectorsAndActions;
+
+    beforeEach(() => {
+      renderAndOpenHelpPanel = (props) => {
+        render(props);
+        helpPanelSelectors = getHelpSidePanelSelectorsAndActionsMock(renderResult);
+        consoleSelectors.openHelpPanel();
+
+        return renderResult;
+      };
+    });
+
+    it('should display the help panel header', () => {
+      renderAndOpenHelpPanel();
+
+      expect(renderResult.getByTestId('test-sidePanel-header')).toHaveTextContent(
+        'HelpUse the add () button to populate a response action to the text bar. Add ' +
+          'additional parameters or comments as necessary.'
+      );
+    });
+
+    it('should display the command list', () => {
+      renderAndOpenHelpPanel();
+
+      expect(renderResult.getByTestId('test-commandList')).toBeTruthy();
+    });
+
+    it('should close the side panel when close button is clicked', () => {
+      renderAndOpenHelpPanel();
+      consoleSelectors.closeHelpPanel();
+
+      expect(renderResult.queryByTestId('test-sidePanel')).toBeNull();
+    });
+
+    it('should display helpful tips', () => {
+      renderAndOpenHelpPanel();
+
+      expect(renderResult.getByTestId('test-commandList-helpfulTips')).toHaveTextContent(
+        'Helpful tips:You can enter consecutive response actions — no need to wait for previous ' +
```

Comment:
What do you think if we put these and the original texts https://github.com/elastic/kibana/blob/53bf25428fa338b3d724ecc106d968f00952cb40/x-pack/plugins/security_solution/public/management/components/console/components/command_list.tsx#L250 as const somewhere, or ideally as i18n. Then we wouldn't have to pay attention to tests when changing the test, or vice versa.

Reply:
I would like to merge this in prior to the next RC testing cycle. I will leave it as is for now and will make the change in a subsequent PR.

Reply:
👍  sure, that makes sense :) Thank you